supervisor/mounts/mount.py

```diff
@@ -197,6 +201,20 @@ async def update(self) -> None:
         ):
             self.sys_resolution.dismiss_issue(self.failed_issue)
 
+    async def _update_state_await(self, expected_state: UnitActiveState) -> None:
+        """Update state info about mount from dbus. Wait up to 30 seconds for the state to appear."""
```

Comment:
Since we're doing exponential waits now, this comment is incorrect. I could be up to about a minute

Reply:
```
>>> total = 0
>>> for i in range(5):
...     total = total + i**2
... 
>>> print(total)
30
```